synapse/cortex.py

```diff
@@ -862,6 +864,13 @@ class Cortex(s_cell.Cell):
         form = self.model.form(name)
         return form.getWaitFor(valu)
 
+    def _initCoreApiCmds(self):
+        '''
+        Registration for cmds types that cmdr is aware of
+        '''
+        self.addCellCmdType('cron')
```

Comment:
since a cortex is the only place that cron lives ( and is specifically tied to storm ) we should just use the cortex interface to load cron comands etc